index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <title>Document</title>
+</head>
+
+<body>
+
+  <header>
+    <nav>
+      <a href="#"><img src="images/MyLogo.png" alt="Logo"></a>
+      <a href="#"> <img src="images/hamb.png" alt="hamb" class="humb"> </a>
+    </nav>
+  </header>
+  <section class="intro">
+    <div class="head">
+      <h1>
+        I’m Lumato
+        Glad to see you!
+      </h1>
+    </div>
+
+    <section class="content">
+      <p>I’m a software developer! I can help you build a
+        product , feature or website Look through some
+        ce! If you like what you see and have a project you need code
+        d, don’t hestiate to contact me.</p>
+      <p class="connect">LETS CONNECT</p>
```

Comment:
**OPTIONAL**
Don't you think it's better and makes more sense if this text is wrapped inside a `<a>` tag instead of `<p>` tag since it's only for paragraphs! Kindly consider this one 🤗